_field-types/mapping-parameters/index-options.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: default
+title: Index_options
+parent: Mapping parameters
+grand_parent: Mapping and field types
+nav_order: 70
+has_children: false
+has_toc: false
+---
+
+# index_options
+
+The `index_options` mapping parameter controls the level of detail stored in the inverted index for text fields. This setting directly influences both the index size and the capabilities available for scoring, phrase matching, and highlighting.
+
+Depending on your requirements, you can choose from the following options:
+
+- `docs`:
+  Only the existence of a term is indexed. No frequency or position information is stored. This option minimizes index size and is sufficient when you only need to confirm that a term appears in a document.
+
+- `freqs`:
+  In addition to recording the existence of terms, the index stores the frequency (the number of times a term appears in a document). This can improve relevance scoring but does not include positional data.
```

Comment:
Freqs stores both docs and freqs.

Positions stores positions, freqs and docs.

Offsets stores offsets, positions, freqs and docs.

Can we clearly add this info as well ?